### PR TITLE
Hybrid plumes for SRBs

### PR DESCRIPTION
**Pull request by vevladdd**
Adds hybrid wfss plumes for the following SRBs:
Algols IB and II (inline and radial)
Castors 1, 2, 4, 4AXL and 120 (inline and radial)
AJ-260 FL and SL (inline and radial)
AJ-60
GEMs 40, 46, 60, 63 (ground-lit and air-lit)
25KS18000 and 18KS7800

### CODE REVIEW

**Comment by StonesmileGit** [comment]
All of the work in this PR is to apply existing templates to the SRB parts. If anything about how the plume looks should be changed then the templates, or the configuration tool needs to change.

**Comment by al2me6** [comment]
> I've noticed that it is not fully aligned with the plume itself, making it seem as if it was coming out the core of the rocket, and it is a bit "discontinued", for lack of a better term.

This is not unexpected, as I alluded to on Discord. 

The fundamental issue is that as the plume gets smaller and thus the individual particles get smaller, the particles take up less space. This means more particles are needed to 'fill' the same volume of space, but the template doesn't account for that. However, the max number of particles is capped. No way around that. 

As @StonesmileGit mentioned, it should be fixable from the template side, at least to some extent. To _what_ extent, I don't know.

File: GameData/ROEngines/Waterfall/SRM/AJ260.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        template = rowaterfall-hybrid-srm-1
+        useHybrid = true
+        audio = srm-2
+        position = 0,0,0.75
+        rotation = 0, 0, 0
+        scale = 4.9, 4.9, 5
+        useRelativeScaling = true
+        glow = ro-srm
+        glowStretch = 0.1
+		ExtraTemplate
```
[the remark this answers is not in the document]
Comment (vevladdd) [reply GameData/ROEngines/Waterfall/SRM/AJ260.cfg]: Done. Is it correct now?

**Comment by NathanKell** [comment]
I believe that this is better than what currently exists, and am therefore merging. Can always improve later if folks have time.

**Pull request merged**